blocks/init/components/button/components/button-options.js

```diff
@@ -0,0 +1,97 @@
+              {__('Button Color', 'eightshift-boilerplate')}
+            </Fragment>
+          }
+          help={__('Change Button Background color.', 'eightshift-boilerplate')}
+          colors={[
+            globalSettings.colors.primary,
+            globalSettings.colors.black,
+          ]}
+          value={styleColor}
+          onChange={onChangeStyleColor}
+        />
+      }
+
+      {onChangeStyleSize &&
+        <SelectControl
+          label={__('Button Size', 'eightshift-boilerplate')}
+          value={styleSize}
+          options={[
+            { label: __('Default', 'eightshift-boilerplate'), value: 'default' },
+            { label: __('Big', 'eightshift-boilerplate'), value: 'big' },
+          ]}
+          onChange={onChangeStyleSize}
+        />
+      }
+
+      {onChangeStyleSizeWidth &&
+        <SelectControl
+          label={__('Button Size Width', 'eightshift-boilerplate')}
+          value={styleSizeWidth}
+          options={[
+            { label: __('Default', 'eightshift-boilerplate'), value: 'default' },
+            { label: __('Block', 'eightshift-boilerplate'), value: 'block' },
+          ]}
+          onChange={onChangeStyleSizeWidth}
+        />
+      }
+
+      {onChangeUrl &&
+        <div className="components-base-control">
+          <label className="components-base-control__label" htmlFor="url">{__('Button Link', 'eightshift-boilerplate')}</label>
+          <URLInput
+            value={url}
+            onChange={onChangeUrl}
+          />
+        </div>
+      }
+
+      {onChangeTitle &&
+        <TextControl
+          label={__('Button Title', 'eightshift-boilerplate')}
```

Comment:
Maybe rename the `title` attribute to `text` attribute.
This is usually called button text, not button title.